foxglove/cmd/info.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/foxglove/foxglove-cli/foxglove/console"
+
+	"github.com/spf13/cobra"
+)
+
+func executeInfo(baseURL, clientID, token, userAgent string) error {
+	isUsingApiKey, err := AuthIsApiKey()
+	if err != nil {
+		return err
+	}
+	if isUsingApiKey {
+		dief("Command not available for auth with api keys")
+	}
+	client := console.NewRemoteFoxgloveClient(baseURL, clientID, token, userAgent)
+	me, err := client.Me()
+	if err != nil {
+		return err
+	}
+	w := tabwriter.NewWriter(os.Stdout, 0, 1, 1, ' ', 0)
+	fmt.Fprintln(w, "Email\t", me.Email)
```

Comment:
Maybe also print something like "Authenticated with session token" here

Reply:
we already have a tablewriter in this repo. Can we use or extend that one instead of adding a new one?